### ISSUE TITLE
tdd-assert does not say where the assertion actually failed

### ISSUE THREAD

**Issue by randie** [nzakas/tdd-assert#1]
When an assertion fails, it would be helpful if tdd-assert tells me where in my code the assertion actually failed (see screenshots below). It tells me where in tdd-assert.js it fails, and where in my code I require tdd-assert, but nothing about the location of the actual assertion failure itself. Please advise.

screenshots:
http://cl.ly/image/0k1a2U0u3f09
http://cl.ly/image/0G3B3n1e2f16

Otherwise, nice module. Thanks for sharing.

**Comment by nzakas** [comment]
I've been looking a bit into how Node itself manages stack traces, I may
just end up pulling in code from there to deal with. I'm assuming that if
you use the native assert module, that you get the info you're looking for?

**Comment by randie** [comment]
Yes, using the native assert module tells me where in my code an assertion failed - http://cl.ly/image/1x3D380n1X3D
